src/Core/Meta/MetaDataProviderInterface.php

```diff
@@ -39,4 +39,36 @@ interface MetaDataProviderInterface
      * @return int
      */
     public function getIdByPage($pageName);
+
+    /**
+     * Gets default page by  meta id.
+     *
+     * @param int $metaId
+     *
+     * @return string
+     */
+    public function getDefaultPageById($metaId);
+
+    /**
+     * Gets module page by meta id.
+     *
+     * @param int $metaId
+     *
+     * @return string
+     */
+    public function getModulePageById($metaId);
+
+    /**
+     * Gets default pages which are not configured in Seo & urls page.
+     *
+     * @return array
+     */
+    public function getDefaultPagesExcludingFilled();
```

Comment:
```suggestion
    public function getDefaultPagesNotConfigured();
```
I find your phpDoc very clear 😄 `Gets default pages which are not configured in Seo & urls page` so why not reuse it for the function name ?